src/Symfony/Component/Security/Core/Exception/EncryptionException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Exception;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class EncryptionException extends \Exception implements EncryptionExceptionInterface
```

Comment:
We tend to avoid extending `\Exception` directly, any other more precise exception class we can use? Maybe `\RuntimeException`

Reply:
Thank you